docs/topics/yaml-rules.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
```

Comment:
Have you checked your preview?

You have not included this module in the assemby